cmd/archivistactl/cmd/bundle.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2025 The Archivista Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/in-toto/archivista/pkg/api"
+	"github.com/in-toto/archivista/pkg/sigstorebundle"
+	"github.com/spf13/cobra"
+)
+
+var bundleCmd = &cobra.Command{
+	Use:   "bundle",
+	Short: "Manage Sigstore bundles",
+}
+
+var importBundleCmd = &cobra.Command{
+	Use:          "import [file]",
+	Short:        "Import a Sigstore bundle",
+	SilenceUsage: true,
+	Args:         cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		filePath := args[0]
+		if gitoid, err := importBundleByPath(cmd.Context(), archivistaUrl, filePath); err != nil {
+			return fmt.Errorf("failed to import bundle %s: %w", filePath, err)
+		} else {
+			rootCmd.Printf("%s imported with gitoid %s\n", filePath, gitoid)
+		}
+		return nil
+	},
+}
+
+var exportBundleCmd = &cobra.Command{
```

Comment:
Is the DSSE ID the same as the gitoid? If not, this will likely return a 404 in all cases.